### DISCUSSION TITLE
Mods

### DISCUSSION THREAD

**Discussion by Floffy02** [QuestCraftPlusPlus/QuestCraft#254] [Q&A]
I'm just checking does this allow mods onto my vr?

**Comment by walksanatora** [comment]
There is a list of mods here
[In this spreadsheet](https://docs.google.com/spreadsheets/d/16SopyOPDz3OrL4qcWwMPzL9s_5Uhl5Q-aRIePY6ztSA/edit?usp=sharing) that is linked in discord

**Reply by Floffy02** [reply]
Ok thanks so do I just download them on my oculus? Or is there a video to watch

**Reply by veneficium42** [reply]
Just put the compatible mods in the mods folder of QuestCraft

**Reply by Floffy02** [reply]
Trough my oculus?

**Reply by Floffy02** [reply]
ok so i cant find .minecraft

**Reply by wildwaga** [reply]
you put the mods in sdcard/Android/data/net.kdt.pojavlaunch.debug/files/.minecraft/mods

**Reply by Tman564ws1** [reply]
If you've side loaded it, you could sideload a third-party File manager Then just download all the mods. Open it up and then put it into the mods folder.
